web/src/main/webapp/js/instructions.js

```diff
@@ -91,6 +89,27 @@ module.exports.create = function (mapLayer, path, urlForHistory, request) {
         instructionsElement.append(moreDiv);
     }
 
+    if (debugDetails) {
+        var detailObj = path.details;
+        for (var detailKey in detailObj) {
+            var pathDetailObj = detailObj[detailKey]["details"];
+            for (var k in pathDetailObj) {
+                var intervalArr = pathDetailObj[k];
+                for (var intervalIndex in intervalArr) {
+                    var interval = intervalArr[intervalIndex];
+                    var lngLat = path.points.coordinates[interval[0]];
+                    L.marker([lngLat[1], lngLat[0]], {
+                        icon: L.icon({
+                            iconUrl: './img/marker-small-blue.png',
+                            iconSize: [15, 15]
+                        }),
+                        draggable: true
+                    }).addTo(mapLayer.getRoutingLayer()).bindPopup(detailKey+":"+k);
+                }
+            }
+        }
+    }
```

Comment:
What do you mean by 2 parameters?

Requesting the details works like this: `path.average_speed=true`. The reason for this is, that it works well with putting this into the hints. If we want something like that, we would need to add it as an actual field to the GHRequest. I thought this might not be worth the overhead, I can do this thought if you like?